requests/add-libcudss-output.yaml

```diff
@@ -0,0 +1,3 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - libcudss: "libcudss*"
```

Comment:
```suggestion
  - libcudss: "libcudss-*"
```

Reply:
"libcudss" is not yet in the namespace, so this is not what we want.

Reply:
Ah yes. The current PR is fine then. Sorry for the noise!